talentmap_api/fsbid/services/admin_panel.py

```diff
@@ -86,46 +88,177 @@ def success_mapping(x):
+
+
+def edit_pmd_mapping(query, date, original_data):
+    mapped_query = {
+        "pmdpmseqnum": query.get("pmdpmseqnum"),
+        "pmdmdtcode": original_data.get("mdt_code"),
+        "pmddttm": date.replace("T", " "),
+        "pmdupdatedate": original_data.get("pmd_update_date", "").replace("T", " "),
+        "pmdupdateid": query.get("hru_id"),
+        "pmdcreatedate": original_data.get("pmd_create_date", "").replace("T", " "),
+        "pmdcreateid": original_data.get("pmd_create_id"),
+    }
+    return mapped_query
+
+
+def create_panel_meeting(query, jwt_token):
     args = {
-        "proc_name": 'act_modPnlMeet',
-        "package_name": 'PKG_WEBAPI_WRAP_SPRINT99',
-        "request_mapping_function": edit_panel_meeting_req_mapping,
-        "response_mapping_function": edit_panel_meeting_res_mapping,
+        "uri": "v1/panels/meeting",
+        "query": query,
+        "query_mapping_function": convert_panel_meeting_create_query,
         "jwt_token": jwt_token,
-        "request_body": data,
+        "mapping_function": None,
     }
-    return services.send_post_back_office(
+
+    return services.get_results_with_post(
         **args
     )
 
-def edit_panel_meeting_req_mapping(request):
-    return {
-        "I_PMT_CODE": "",
-        "I_PMS_CODE ": "",
-        "I_PM_DELETE_IND": "",
-        "I_PM_SEQ_NUM": "",
-        "I_PM_VIRTUAL_IND": "",
-        "I_PM_UPDATE_ID": "",
-        "I_PM_UPDATE_DATE": "",
-        "I_INC_IND": "",
-        "I_PMD_DTTM": "",
-        "I_MDT_CODE": "",
-        "I_PMD_UPDATE_ID": "",
-        "I_PMD_UPDATE_DATE": ""
+
+def create_panel_meeting_date(query, date, date_type, jwt_token):
+    pmseqnum = query.get("pmdpmseqnum")
+    args = {
+        "uri": f"v1/panels/meeting/{pmseqnum}/dates",
+        "query": query,
+        "query_mapping_function": partial(create_pmd_mapping, date=date, date_type=date_type),
+        "jwt_token": jwt_token,
+        "mapping_function": None,
+    }
+
+    return services.get_results_with_post(
+        **args
+    )
+
+
+def convert_panel_meeting_create_query(query):
+    hru_id = query.get("hru_id")
+    mapped_query = {
+        "pmvirtualind": "N",
+        "pmcreateid": hru_id,
+        "pmupdateid": hru_id,
+        "pmpmscode": query.get("panelMeetingStatus", "I"),
+        "pmpmtcode": query.get("panelMeetingType", "ID"),
+    }
+    return mapped_query
+
+
+def create_pmd_mapping(query, date, date_type):
+    hru_id = query.get("hru_id")
+    mapped_query = {
+        "pmdpmseqnum": query.get("pmdpmseqnum"),
+        "pmdmdtcode": date_type,
+        "pmddttm": date.replace("T", " "),
+        "pmdupdateid": hru_id,
+        "pmdcreateid": hru_id,
     }
+    return mapped_query
+
+
+
+# ======================== Edit Panel Meeting ========================
+def edit_panel_meeting(query, jwt_token):
+    pm_seq_num = query.get("originalReference", {}).get("pm_seq_num")
+    args = {
+        "uri": f"v1/panels/meeting/{pm_seq_num}",
```

Comment:
```suggestion
    pmseqnum = query.get("originalReference", {}).get("pm_seq_num")
    args = {
        "uri": f"v1/panels/meeting/{pmseqnum}",
```

just for consistency